http-client/http-client.cabal

```diff
@@ -1,5 +1,5 @@
 name:                http-client
-version:             0.7.12
+version:             0.7.12.1
```

Comment:
This would be a minor version bump to 0.7.13, and require a ChangeLog entry too.